src/Evolution/Systems/GeneralizedHarmonic/Tags.hpp

```diff
@@ -117,5 +172,52 @@ struct EvolvedFieldsFromCharacteristicFields : db::SimpleTag {
     return "EvolvedFieldsFromCharacteristicFields";
   }
 };
+
+/*!
+ * \ingroup GeneralizedHarmonicGroup
+ * \brief Gauge control parameter determining when to start rolling-on the
+ * evolution gauge.
+ *
+ * \details The evolution gauge is gradually transitioned to (or
+ * *rolled-on* to) at the beginning of an evolution. This parameter sets
+ * the coordinate time at which roll-on begins.
+ */
+struct GaugeHRollOnStartTime : db::SimpleTag {
```

Comment:
Please move this and similar tags into a `GeneralizedHarmonic::OptionTags` namespace, since it will be read from an input file. Add an `OptionString help` and make sure that it and the `name`s are short enough by adding a test that constructs these from options.

Reply:
moved, test of optionTags added.